### ISSUE TITLE
openbsd64 box broken: no route to host

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#2042]
```
==> openbsd64: Can't find lz4
==> openbsd64: Error from http://openbsd.cs.fau.de/pub/OpenBSD/6.0/packages/amd64/
==> openbsd64: ftp: connect: No route to host
==> openbsd64: http://openbsd.cs.fau.de/pub/OpenBSD/6.0/packages/amd64/ is empty
==> openbsd64: Can't find git
```

**Comment by ThomasWaldmann** [comment]
/etc/pkg.conf:
```
installpath = http://openbsd.cs.fau.de/pub/OpenBSD/6.0/packages/amd64/
```
... which does not exist.

**Comment by ThomasWaldmann** [comment]
installpath = http://ftp.hostserver.de/pub/OpenBSD/6.0/packages/amd64/
works

**Comment by ThomasWaldmann** [comment]
```
diff --git a/Vagrantfile b/Vagrantfile
index 1fe981f..4dfb897 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -111,6 +111,8 @@ end
 
 def packages_openbsd
   return <<-EOF
+    echo 'installpath = http://ftp.hostserver.de/pub/OpenBSD/6.0/packages/amd64/' > /etc/pkg.conf
+    echo 'export PKG_PATH=http://ftp.hostserver.de/pub/OpenBSD/6.0/packages/amd64/' >> ~/.profile
     . ~/.profile
     pkg_add bash
     chsh -s /usr/local/bin/bash vagrant
```

**Comment by enkore** [comment]
Fixed by #2044
